test/mqtt_ascoltatore_spec.js

```diff
@@ -1,3 +1,70 @@
+it("should re-subscribe to topics", function(done) {
```

Comment:
I can't use that global as I need to be able to start and stop it.  It seems that multiple iterations of test are invoked if I put my re-register test inside of the "describe".  I get the following error:

  1) ascoltatori.MQTTAscoltatoreDifferentWildcardAndSeparator should re-subscribe to topics:
     Uncaught Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1129:14)
      at listen (net.js:1155:10)
      at net.js:1253:9
      at dns.js:85:18
      at process._tickDomainCallback (node.js:381:11)

  2) ascoltatori.MQTTAscoltatoreDifferentWildcardAndSeparator "after each" hook:
     Uncaught Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1129:14)
      at listen (net.js:1155:10)
      at net.js:1253:9
      at dns.js:85:18
      at process._tickDomainCallback (node.js:381:11)